study-group/README.md

```diff
@@ -0,0 +1,130 @@
+This list is by no means exhaustive, but it covers many of the core elements built into popular toast libraries. 
+They approximately break down into three categories: Properties, Built-In Support, and Policies.
+
+### Properties
+Properties of a toast are specific, tangible parts of the object that a developer can set, toggle, or access.
+These are often the user-facing elements of the toast, and the general consensus of popular libraries regarding these properties is typically what sets toasts apart from other solutions like alerts and dialogs.
+
+- `title`: header text to be displayed above the main message of the toast. 
+
+- `message`: main text contents of the toast. 
+
+- `timeout`: time period (usually in ms) after which the toast disappears.
+
+- Close Button: dedicated button that closes the toast when clicked.
+
+- Call To Action: button(s) displayed on the toast prompting user engagement.
+
+- Callbacks: functions triggered when the toast enters or exits certain states.
+
+- Events: JavaScript events triggered when the toast enters or exits certain states.
+
+- `icon`: pictograph displayed with the toast.
+
+- Progress Bar: element on toast displaying time remaining before timeout.
+
+### Built-In Support
+The Built-In Support establishes how a toast is defined and used, as well as what it's allowed to be used with.
+It consists of a collection of implementation details, patterns, and technologies that are supported by popular libraries.
+
+- Theme Support: offers built-in theming options, such as `success`, `info`, or `error`.
+
+- Global Custom Configuration: support for global config objects defining custom themes / patterns, which can be applied to individual toasts.
+
+- Configured / Created in JS: the toast object is set up and called to display using JavaScript.
+    - Upon display this creates a new DOM element or injects the settings into an existing HTML element.
+
+- Explicit Component Pattern: uses an element explicitly written into the HTML to display the toast when triggered.
+    - This element can be configured, or left as a dummy element to inject configuration into.
+
+- Permits Custom HTML: allows developers to write custom HTML templates, loaded in as children or passed as views.
+
+- Display in Container: allows the toast to display within a container, the default being `body`.
+
+- CSS Class Field: explicit field to pass in the CSS class or list of classes for styling the toast.
+
+- Explicit Animation Support: library offers an `animation` or `transition` field which abstracts away writing animations.
+
+- Swipe / Drag: the toast can be swiped or dragged to be dismissed, an important cross-platform consideration.
+
+- Intuitive Positioning: the library provides built in values to place your toast on the `top/middle/bottom` of the `left/center/right`.
```

Comment:
```suggestion
- Intuitive Positioning: the library provides built in values to place your toast on the `top/middle/bottom` or the `left/center/right`.
```

Reply:
Do you think it might be better as: 
```suggestion
- Intuitive Positioning: the library provides built in values to place your toast on the `top/middle/bottom` and the `left/center/right`.
 ```
since the vertical and horizontal positions are can be used together?

Reply:
Yeah, that seems better!